shared/src/utils/arrayDisplayCodeBlock.ts

```diff
@@ -7,32 +7,81 @@ import { SingleTranslation } from "@shared/utils/translations"
  * @param startingIndex The index to in the top row to start with
  *                      (default is 0, mostly either 0 or 1)
  * @param secondRowName The name of the second row (default is ST "Value")
+ * @param transposeMobile If true, the array will be displayed transposed on smaller devices
  */
 export type ArrayDisplayProps<T> = {
   array: T[]
   startingIndex: number
   secondRowName: SingleTranslation
+  transposeMobile: boolean
+  inputFieldSize?: number
+}
+
+/**
+ * This function creates an array display code block with user input fields.
+ * @param numberOfInputFields - (more than nine input fields may be too large)
+ * @param startingIndex
+ * @param secondRowName
+ * @param transposeMobile
+ * @param inputFieldSize - the size of the input field (11 is the default size (it fits two digits))
+ *                         (currently only **w-(10|11|12|14)** are included in the safelist,
+ *                         in case you need a different size, please add it to the safelist)
```

Comment:
Ich würde die Breite des Textfeldes lieber in der Anzahl der benötigten Buchstaben angeben, das Front-end entscheidet dann, was es damit macht.